common/archiver/provider/provider.go

```diff
@@ -25,20 +25,21 @@ import (
 
 	"github.com/uber/cadence/common/archiver"
 	"github.com/uber/cadence/common/archiver/filestore"
-	"github.com/uber/cadence/common/log/tag"
 )
 
 var (
 	// ErrUnknownScheme is the error for unknown archiver scheme
 	ErrUnknownScheme = errors.New("unknown archiver scheme")
+	// ErrEmptyBootStrapContainer is the error for empty bootstrap container
+	ErrEmptyBootStrapContainer = errors.New("empty bootstrap container")
```

Comment:
"empty bootstrap container can only be provided if archiver has already been created"

Reply:
Since now we have the create method,  I will keep the current error message as it should always be not nil.